AppContent null?

Google play console:
```
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.ui.activity.DrawerActivity.onCreate (DrawerActivity.java:1143)
  at com.owncloud.android.ui.activity.FileActivity.onCreate (FileActivity.java:148)
  at com.owncloud.android.ui.activity.FileDisplayActivity.onCreate (FileDisplayActivity.java:186)
```
which is first line of:
https://github.com/nextcloud/android/blob/314526005489943cf8a35d8c4ace2f4db802922a/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.java#L1143-L1144

We could change the first line to be like second line, but I do not know if this is  then crashing later, if contentResolver is null.

Unfortunately I cannot reproduce this :/

--- a/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.java
+++ b/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.java
@@ -1140,7 +1140,7 @@ public abstract class DrawerActivity extends ToolbarActivity implements DisplayU
         mMenuAccountAvatarRadiusDimension = getResources()
                 .getDimension(R.dimen.nav_drawer_menu_avatar_radius);
 
-        externalLinksProvider = new ExternalLinksProvider(MainApp.getAppContext().getContentResolver());
+        externalLinksProvider = new ExternalLinksProvider(getContentResolver());
         arbitraryDataProvider = new ArbitraryDataProvider(getContentResolver());
     }
 

